### PR TITLE
Confg subscriber init tests

### PR DESCRIPTION
**Pull request by iamigo**

### CODE REVIEW

File: test/config.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+      beforeEach(() => uncache('../conf/config'));
+
+      it('not defined', () => {
+        const config = require('../conf/config');
+        expect(config.pubSubPerspectives).to.deep.eql([]);
+      });
+
+      it('empty string', () => {
+        const config = require('../conf/config');
+        expect(config.pubSubPerspectives).to.deep.eql([]);
```
Comment (jgraff2) [inline test/config.js]: not actually setting the env var here.

File: test/config.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        process.env.REDIS_PUBSUB_PERSPECTIVES = 'SOMETHING';
+        const config = require('../conf/config');
+        expect(config.pubSubPerspectives).to.deep.eql([]);
+      });
+
+      it('has corresponding env var', () => {
+        process.env.REDIS_PUBSUB_PERSPECTIVES = 'SOMETHING';
+        process.env.SOMETHING = 'abcdefg';
+        const config = require('../conf/config');
+        expect(config.pubSubPerspectives).to.deep.eql([process.env.SOMETHING]);
```
Comment (jgraff2) [inline test/config.js]: would be clearer if it expected ['abcdefg'] explicitly.

File: test/config.js [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+      beforeEach(() => uncache('../conf/config'));
+
+      it('not defined', () => {
+        const config = require('../conf/config');
+        expect(config.pubSubBots).to.deep.eql([]);
+      });
+
+      it('empty string', () => {
+        const config = require('../conf/config');
+        expect(config.pubSubBots).to.deep.eql([]);
```
Comment (jgraff2) [inline test/config.js]: not actually setting the env var here.

File: .travis.yml [hunk 4]
```diff
@@ -23,6 +23,9 @@ env:
   global:
     - NODE_ENV=build
 
+services:
+  - redis-server
+
```
[the remark this answers is not in the document]
Comment (iamigo) [reply .travis.yml]: Not really a big plan... I like the idea of starting to leverage heroku-ci for some of these "smaller" applications... but not ready to abandon travis either

File: test/config.js (hunk 2 above)
Comment (jgraff2) [reply test/config.js]: same for the other tests

**Pull request merged**